src/views/tls/tls-version/index.js

```diff
@@ -18,33 +22,58 @@ import React, {useEffect, useState} from 'react';
  *
  * @return {ReactNode} TLS versions component
  */
+
 const TLSversion = () => {
   const [loading, setLoading] = useState(true);
+  const [empty, setEmpty] = useState(false);
   const [snackOpen, setSnackOpen] = useState(false);
   const [data, setData] = useState([]);
-  const [table, setTable] = useState(
-    'httparchive.sample_data.requests_desktop_10k',
-  );
+  const [year, setYear] = useState('sample');
+  const [month, setMonth] = useState('01');
+  const months = [
+    '01',
+    '02',
+    '03',
+    '04',
+    '05',
+    '06',
+    '07',
+    '08',
+    '09',
+    '10',
+    '11',
+    '12',
+  ];
 
   useEffect(() => {
     setLoading(true);
     api
-      .get('/api/tls/tls-version?table=' + table)
+      .get('/api/tls/tls-version?year=' + year + '&month=' + month)
       .then((response) => {
         setLoading(false);
         setData(response.data.result);
+        if (response.data.result.length < 1) {
+          setEmpty(true);
+        } else {
+          setEmpty(false);
+        }
       })
       .catch((err) => {
         setSnackOpen(true);
       });
-  }, [table]);
+    console.log(empty);
```

Comment:
Should these console.log statements be deleted?

Reply:
Thanks! Just deleted this